crates/clients/chainbound/src/mev_bundle.rs

```diff
@@ -48,6 +50,7 @@ impl ToString for BlockBuilder {
             BlockBuilder::Blocknative => "blocknative".to_string(),
             BlockBuilder::Nfactorial => "nfactorial".to_string(),
             BlockBuilder::Buildai => "buildai".to_string(),
+            BlockBuilder::Loki => "lokibuilder".to_string(),
```

Comment:
FYI, we named the builder "loki" as in [our documentation](https://echo.chainbound.io/docs/usage/integrations).

```suggestion
            BlockBuilder::Loki => "loki".to_string(),
```

Reply:
Thanks! Updated